include/deal.II/multigrid/mg_tools.h

```diff
@@ -247,6 +247,18 @@ namespace MGTools
   void
   extract_non_interface_dofs (const DoFHandler<dim,spacedim> &mg_dof_handler,
                               std::vector<std::set<types::global_dof_index> > &non_interface_dofs);
+
+
+  /**
+   * Return the highest possible level that can be used as the coarsest level in
+   * a Multigrid computation, that is, the highest level in the hierarchy whose mesh
+   * covers the entire domain. This corresponds to the minimum level of a cell on
+   * the active mesh. Note that this is a global minimum over each processor and therefore
+   * each processor will return the same value.
+   */
+  template <int dim, int spacedim>
+  unsigned int
+  max_level_for_coarse_mesh (const Triangulation<dim,spacedim> &tria);
```

Comment:
I know it's kind of implied, but I would actively say that the call is collective over the communicator in the triangulation and needs to be called by all processors.